build.sh

```diff
@@ -0,0 +1 @@
+make
```

Comment:
I don't mind this, but what is this for?

Reply:
oh this was so i could run make within intellij, which required a shell script.. 

i removed it from the repo and just gitignored it